test/functional/c/CMakeLists.txt

```diff
@@ -1,16 +1,16 @@
 list(APPEND ctest_labels c_api)
 set(test_sources
-		test_delaunay.cpp
-		test_error.cpp
-		test_symmetry.cpp
-		test_symmetry_search.cpp
-		test_spacegroup_type_search.cpp
-		test_find_primitive_cell.cpp
-		test_refine_cell.cpp
-		test_standardization.cpp
-		test_dataset_access.cpp
-		test_magnetic_symmetry.cpp
-		test_kpoints.cpp
+		delaunay.cpp
+		error.cpp
+		symmetry.cpp
+		symmetry_search.cpp
+		spacegroup_type_search.cpp
+		find_primitive_cell.cpp
+		refine_cell.cpp
+		standardization.cpp
+		dataset_access.cpp
+		magnetic_symmetry.cpp
+		kpoints.cpp
```

Comment:
Why do you rename `test_kpoints.cpp` to `kpoints.cpp` and so on? Is there any correspoding style guide? If not, I prefer not to change these file names.

Reply:
Not really, it's just more consistent with the naming of the test-suite. But I do get the appeal that it separates the tests from the unity libraries